Music app/app/src/main/java/com/example/musicapp/NotificationPlaying/ApplicationClass.java

```diff
@@ -0,0 +1,38 @@
+package com.example.musicapp.NotificationPlaying;
+
+import android.app.Application;
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.os.Build;
+
+public class ApplicationClass extends Application
```

Comment:
For what is this class?

Reply:
It's for the notification channel. In one android app - to show notifications you need channels. With that channels, you create connection and synchronization between activities in your application and notifications of the phone.

Reply:
Why you use two channels?

Reply:
I can't remember why there had to be two, but NotificationManager just requires it